releasenotes/notes/pylons-int-config-b500a59f78297b88.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    Added an environment variable to enable distributed tracing for pylons.
```

Comment:
lol sorry for not being more clear with the previous review 🙈  I was thinking that we'd explicitly state the environment variable: `PYLONS_DISTRIBUTED_TRACING` else the reader of the release notes would have to go digging into the code to find out what it was

Reply:
We should also add it to the integration docs in `pylons/__init__.py`